docs/guides/qpu-information.ipynb

```diff
@@ -0,0 +1,805 @@
+    "\n",
+    "        The following code returns only backends that fit these criteria and are available to you _on your currently selected instance_:\n",
+    "\n",
+    "        *   Are real quantum devices (`simulator=False`)\n",
+    "        *   Are currently operational (`operational=True`)\n",
+    "        *   Have at least a hundred qubits (`min_num_qubits=100`)\n",
+    "\n",
+    "        <CodeCellPlaceholder tag=\"id-filter\" />\n",
+    "\n",
+    "        A similar method is [`QiskitRuntimeService.least_busy()`](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service#least_busy), which takes the same filters as `backends()` but returns the backend that matches the filters and has the least number of jobs pending in the queue:\n",
+    "\n",
+    "        <CodeCellPlaceholder tag=\"id-least-jobs\" />\n",
+    "\n",
+    "    </TabItem>\n",
+    "    <TabItem value=\"IQP-filter\" label=\"Filter by properties on IBM Quantum Platform\">\n",
+    "        To view the backends you have access to, view the table on the [Compute resources page](https://quantum.cloud.ibm.com/computers). Click the filter icon to see filter options. You can filter by processor type and status. Note also that you can sort the table by any column by hovering over a column title, and then clicking the arrows that appear.\n",
+    "    </TabItem>\n",
+    "</Tabs>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "846dfe4b-def3-49c1-9d2a-906fd2dc55de",
+   "metadata": {
+    "tags": [
+     "id-least-jobs"
+    ]
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<IBMBackend('ibm_fez')>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "service.least_busy(operational=True, min_num_qubits=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b34bb71-b8cf-415f-9901-a5970997ee15",
+   "metadata": {},
+   "source": [
+    "## Static backend information\n",
+    "\n",
+    "Some information about a backend does not change regularly, such as its name, version, the number of qubits it has, and the types of features it supports. This information is available as attributes of the `backend` object. For a full list of attributes, see the [`IBMBackend` API documentation](/docs/api/qiskit-ibm-runtime/ibm-backend).\n",
+    "\n",
+    "### QPU versioning\n",
+    "\n",
+    "Each QPU has a version number in the form X.Y.Z (major.minor.revision). A circuit compiled for a given version number is guaranteed to run on that QPU. If the revision number changes, the circuit will continue to run. If the major or minor number changes, the circuit is not guaranteed to run, although it may do so.\n",
```

Comment:
> A circuit compiled for a given version number is guaranteed to run on that QPU

I don't know if this statement is really true. For example, I'm not sure if the version number changed when we toggled dynamic circuits support. I'll ask in Jan when people are back.